app/models/common/ManualOrUpload.scala

```diff
@@ -14,36 +14,37 @@
  * limitations under the License.
  */
 
-package models.event1
+package models.common
 
+import models.enumeration.EventType
 import models.{Enumerable, WithName}
 import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.Aliases.Text
 import uk.gov.hmrc.govukfrontend.views.viewmodels.hint.Hint
 import uk.gov.hmrc.govukfrontend.views.viewmodels.radios.RadioItem
 
-sealed trait HowAddUnauthPayment
+sealed trait ManualOrUpload
 
-object HowAddUnauthPayment extends Enumerable.Implicits {
+object ManualOrUpload extends Enumerable.Implicits {
 
-  case object Manual extends WithName("manual") with HowAddUnauthPayment
-  case object FileUpload extends WithName("fileUpload") with HowAddUnauthPayment
+  case object Manual extends WithName("manual") with ManualOrUpload
+  case object FileUpload extends WithName("fileUpload") with ManualOrUpload
 
-  val values: Seq[HowAddUnauthPayment] = Seq(
-    Manual, FileUpload
-  )
+  def values: Seq[ManualOrUpload] = {
```

Comment:
Would this not be better (more efficient) as a val?

Reply:
Yes, it would be, I must have forgotten to revert the change from a def.

Reply:
OK, well I merged anyway as very minor and shouldn't matter